<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="AcrylicFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>580</width>
     <height>300</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="settings_school_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>学校</string>
    </property>
   </widget>
   <widget class="QLabel" name="settings_phone_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>电话</string>
    </property>
   </widget>
   <widget class="QLabel" name="settings_name_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>姓名</string>
    </property>
   </widget>
   <widget class="QLabel" name="settings_age_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>年龄</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="settings_name_edit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>35</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="settings_age_edit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>95</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="settings_school_edit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>155</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="settings_phone_edit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>215</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="settings_pic_label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>请上传图片</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="settings_upload_btn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>50</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>上传
图片</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="settings_say_edit">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>150</y>
      <width>210</width>
      <height>100</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
   </widget>
   <widget class="QLabel" name="settings_say_label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>150</y>
      <width>50</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string>备注</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="settings_reload_btn">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>50</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>重置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="settings_check_btn">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>40</y>
      <width>50</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>确认
更改</string>
    </property>
   </widget>
   <widget class="QPushButton" name="settings_exit_btn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>260</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AcrylicFrame</class>
   <extends>QFrame</extends>
   <header location="global">acrylicframe.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
